src/control/server/testdata/ioparams_daos_server_uncomment.txt

```diff
@@ -1 +1 @@
-[{Rank:0 Targets:[1-20] NrXsHelpers:0 FirstCore:1 FabricIface:qib0 FabricIfacePort:20000 LogMask:WARN LogFile:/tmp/daos_server1.log EnvVars:[CRT_TIMEOUT=30 CRT_PHY_ADDR_STR=ofi+verbs;ofi_rxm OFI_INTERFACE=qib0 OFI_PORT=20000 D_LOG_MASK=WARN D_LOG_FILE=/tmp/daos_server1.log] ScmMount:/mnt/daos/1 ScmClass:ram ScmList:[] ScmSize:16 BdevClass:nvme BdevList:[0000:81:00.0] BdevNumber:0 BdevSize:0 CliOpts:[-t 20 -g daos -s /mnt/daos/1 -x 0 -f 1 -r 0 -d ./.daos/daos_server] formatted:<nil>} {Rank:1 Targets:[21-40] NrXsHelpers:1 FirstCore:21 FabricIface:qib0 FabricIfacePort:20000 LogMask:WARN LogFile:/tmp/daos_server2.log EnvVars:[CRT_TIMEOUT=100 CRT_PHY_ADDR_STR=ofi+verbs;ofi_rxm OFI_INTERFACE=qib0 OFI_PORT=20000 D_LOG_MASK=WARN D_LOG_FILE=/tmp/daos_server2.log] ScmMount:/mnt/daos/2 ScmClass:dcpm ScmList:[/dev/pmem0] ScmSize:0 BdevClass:kdev BdevList:[/dev/sdc /dev/sdd] BdevNumber:1 BdevSize:16 CliOpts:[-t 20 -g daos -s /mnt/daos/2 -x 1 -f 21 -r 1 -d ./.daos/daos_server] formatted:<nil>}]
+[{Rank:0 Targets:[1-20] NrXsHelpers:0 FirstCore:1 FabricIface:qib0 FabricIfacePort:20000 LogMask:WARN LogFile:/tmp/daos_server1.log EnvVars:[CRT_TIMEOUT=30 CRT_PHY_ADDR_STR=ofi+verbs;ofi_rxm OFI_INTERFACE=qib0 D_LOG_MASK=WARN D_LOG_FILE=/tmp/daos_server1.log OFI_PORT=20000] ScmMount:/mnt/daos/1 ScmClass:ram ScmList:[] ScmSize:16 BdevClass:nvme BdevList:[0000:81:00.0] BdevNumber:0 BdevSize:0 CliOpts:[-t 20 -g daos -s /mnt/daos/1 -x 0 -f 1 -r 0 -d ./.daos/daos_server] formatted:<nil>} {Rank:1 Targets:[21-40] NrXsHelpers:1 FirstCore:21 FabricIface:qib0 FabricIfacePort:20000 LogMask:WARN LogFile:/tmp/daos_server2.log EnvVars:[CRT_TIMEOUT=100 CRT_PHY_ADDR_STR=ofi+verbs;ofi_rxm OFI_INTERFACE=qib0 D_LOG_MASK=WARN D_LOG_FILE=/tmp/daos_server2.log OFI_PORT=20000] ScmMount:/mnt/daos/2 ScmClass:dcpm ScmList:[/dev/pmem0] ScmSize:0 BdevClass:kdev BdevList:[/dev/sdc /dev/sdd] BdevNumber:1 BdevSize:16 CliOpts:[-t 20 -g daos -s /mnt/daos/2 -x 1 -f 21 -r 1 -d ./.daos/daos_server] formatted:<nil>}]
```

Comment:
i noticed this has OFI_INTERFACE=qib0   -- is 'qib0' (vs ib0) an intentional?

Reply:
I think it's an example value that was provided in one of the f2f meetings a long while ago, it may be a typo but I would have to check with @johannlombardi